app/components/SideBar.js

```diff
@@ -10,6 +10,7 @@ import './fonts.css';
 function mapStateToProps(state) {
   return {
     walletService: state.grpc.walletService,
+    ticketBuyerService: state.grpc.ticketBuyerService,
```

Comment:
I'll remove.  That was when I was getting a little desperate trying to find where I was missing something.